identityClient/identityClient.go

```diff
@@ -1422,7 +1422,7 @@ func (c *E3dbIdentityClient) RegisterMFADevice(ctx context.Context, params Regis
 		return response, err
 	}
 	request.Header.Add(toznySessionHeader, params.SessionToken)
-	err = e3dbClients.MakeSignedServiceCall(ctx, c.requester, request, c.SigningKeys, c.ClientID, response)
+	err = e3dbClients.MakeSignedServiceCall(ctx, c.requester, request, c.SigningKeys, c.ClientID, &response)
```

Comment:
if you update the signature of `MakeSignedServiceCall` to take a pointer to an interface (*interface{}) does the code still compile, and does doing it the old/broken way result in a compile time error?

Reply:
Both result in a compile time error